packages/kbn-rule-data-utils/src/default_alerts_as_data.ts

```diff
@@ -70,6 +70,9 @@ const ALERT_WORKFLOW_STATUS = `${ALERT_NAMESPACE}.workflow_status` as const;
 // kibana.alert.workflow_tags - user workflow alert tags
 const ALERT_WORKFLOW_TAGS = `${ALERT_NAMESPACE}.workflow_tags` as const;
 
+// kibana.alert.evaluation.value - alert evaluation value
+const ALERT_EVALUATION_VALUE = `${ALERT_NAMESPACE}.evaluation.value` as const;
```

Comment:
@kdelemme Look what I found [here](https://github.com/elastic/kibana/blob/main/packages/kbn-rule-data-utils/src/technical_field_names.ts#L89). So let's reuse this one. I searched in the kibana repo and it looks like they use this one.

```
import {
  ALERT_EVALUATION_VALUE,
} from '@kbn/rule-data-utils';
```

Reply:
Oh I see you already import ALERT_EVALUATION_VALUE from '@kbn/rule-data-utils' later on. So maybe no need to export ALERT_EVALUATION_VALUE  in the default alert as data file?